zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/handlers/UserTaskHandler.java

```diff
@@ -0,0 +1,235 @@
+          case "candidateGroupsList" ->
+              updateFields.put(TaskTemplate.CANDIDATE_GROUPS, entity.getCandidateGroups());
+          case "candidateUsersList" ->
+              updateFields.put(TaskTemplate.CANDIDATE_USERS, entity.getCandidateUsers());
+          case "dueDate" -> updateFields.put(TaskTemplate.DUE_DATE, entity.getDueDate());
+          case "followUpDate" ->
+              updateFields.put(TaskTemplate.FOLLOW_UP_DATE, entity.getFollowUpDate());
+          case "priority" -> updateFields.put(TaskTemplate.PRIORITY, entity.getPriority());
+          default ->
+              LOGGER.warn(
+                  "Attribute update not mapped while importing ZEEBE_USER_TASKS: {}",
+                  changedAttribute);
+        }
+      }
+    }
+    if (entity.getCompletionTime() != null) {
+      updateFields.put(TaskTemplate.COMPLETION_TIME, entity.getCompletionTime());
+    }
+
+    if (entity.getChangedAttributes() != null && !entity.getChangedAttributes().isEmpty()) {
+      updateFields.put(TaskTemplate.CHANGED_ATTRIBUTES, entity.getChangedAttributes());
+    }
+    if (entity.getAction() != null) {
+      updateFields.put(TaskTemplate.ACTION, entity.getAction());
+    }
+    if (entity.getState() != null) {
+      updateFields.put(TaskTemplate.STATE, entity.getState());
+    }
+    if (entity.getFlowNodeBpmnId() != null) {
+      updateFields.put(TaskTemplate.FLOW_NODE_BPMN_ID, entity.getFlowNodeBpmnId());
+    }
+    if (entity.getProcessDefinitionId() != null) {
+      updateFields.put(TaskTemplate.PROCESS_DEFINITION_ID, entity.getProcessDefinitionId());
+    }
+    if (entity.getBpmnProcessId() != null) {
+      updateFields.put(TaskTemplate.BPMN_PROCESS_ID, entity.getBpmnProcessId());
+    }
+
+    return updateFields;
+  }
+
+  private void createTaskEntity(final TaskEntity entity, final Record<UserTaskRecordValue> record) {
+    entity
+        .setId(String.valueOf(record.getValue().getElementInstanceKey()))
+        .setKey(record.getKey())
+        .setState(TaskState.CREATED)
+        .setAssignee(
+            ExporterUtil.isEmpty(record.getValue().getAssignee())
+                ? null
+                : record.getValue().getAssignee())
+        .setDueDate(ExporterUtil.toOffsetDateTime(record.getValue().getDueDate()))
+        .setFollowUpDate(ExporterUtil.toOffsetDateTime(record.getValue().getFollowUpDate()))
+        .setFlowNodeInstanceId(String.valueOf(record.getValue().getElementInstanceKey()))
+        .setProcessInstanceId(String.valueOf(record.getValue().getProcessInstanceKey()))
+        .setFlowNodeBpmnId(record.getValue().getElementId())
+        .setBpmnProcessId(record.getValue().getBpmnProcessId())
+        .setProcessDefinitionId(String.valueOf(record.getValue().getProcessDefinitionKey()))
+        .setProcessDefinitionVersion(record.getValue().getProcessDefinitionVersion())
+        .setFormKey(
+            record.getValue().getFormKey() > 0
```

Comment:
[Question❓]  Does the `recordValue` does not return directly null when formKey does not exist?  (I know we have this problem for `externalFormRererence`

Reply:
Zeebe does not set null on numerical fields, just negative values. So the check for > 0 is required here.